site.go

```diff
@@ -182,14 +192,30 @@ func (s *Site) GetS3Photo(key string) *S3Photo {
 	}
 }
 
-func (s *Site) GetImgixPhoto(key string) *ImgixPhoto {
+func (s *Site) GetScaledPhoto(resizingService string, resizingServiceSecret string, key string) Renderable {
```

Comment:
No need to pass the `resizingService` and `resizingServiceSecret` here as that's all part of the site struct.

Reply:
yeah, i was already implementing another resizing backend and will be reconsidering this approach completely, probably just use the site struct... is there any considerations/concerns when exposing the entire site configuration to a `RescaledPhoto` object?

Reply:
Not that I can think of. Since this method returns a `Renderable` interface object, no one who has access to the `Renderable` can get the secret keys, which are part of the struct `ImgixResizedPhoto`/`ThumborResizedPhoto` etc.